bindings/rust/generate/_run_supported_symbol_builds.sh

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env bash
```

Comment:
What is the intended purpose of this script? Can we update our existing CI to run these builds?